vim.txt

```diff
@@ -0,0 +1,10 @@
+[Section]
+Name = Vi IMproved
+Version = 8.2.5171
+License = GPL-compatible charityware
+Description = Vim is a highly configurable text editor built to enable efficient text editing. It is an improved version of the vi editor distributed with most UNIX systems.
+Category = 7
```

Comment:
Not sure about category 7 (Development), since for example Notepad++ uses 6 (Office).

Reply:
I'd argue that Notepad++ is in the wrong category then.